docker-images/bundle/Makefile

```diff
@@ -0,0 +1,70 @@
+PROJECT_NAME=bundle
+
+CRD_DIR := ../../install/cluster-operator
+DEPLOYMENT_FILE := $(CRD_DIR)/060-Deployment-strimzi-cluster-operator.yaml
+MANIFESTS := ./manifests
+CSV_FILE := $(MANIFESTS)/strimzi-cluster-operator.clusterserviceversion.yaml
+DATE := $(shell date +'%Y-%m-%d %H:%M:%S')
+
+CURRENT_BUNDLE_VERSION := $(shell cat ../../release.version)
+PREVIOUS_BUNDLE_VERSION := $(shell curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/strimzi-kafka-operator/strimzi-kafka-operator.package.yaml | yq e '.channels[0].currentCSV' -)
+
+update_bundle:
+	# Copy Custom Resource Definitions (CRDs)
+	$(CP) $(CRD_DIR)/043-Crd-kafkatopic.yaml $(MANIFESTS)/kafkatopics.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/044-Crd-kafkauser.yaml $(MANIFESTS)/kafkausers.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/040-Crd-kafka.yaml $(MANIFESTS)/kafkas.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/041-Crd-kafkaconnect.yaml $(MANIFESTS)/kafkaconnects.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/042-Crd-kafkaconnects2i.yaml $(MANIFESTS)/kafkaconnects2is.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/045-Crd-kafkamirrormaker.yaml $(MANIFESTS)/kafkamirrormakers.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/046-Crd-kafkabridge.yaml $(MANIFESTS)/kafkabridges.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/047-Crd-kafkaconnector.yaml $(MANIFESTS)/kafkaconnectors.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/048-Crd-kafkamirrormaker2.yaml $(MANIFESTS)/kafkamirrormaker2s.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/049-Crd-kafkarebalance.yaml $(MANIFESTS)/kafkarebalances.kafka.strimzi.io.crd.yaml
```

Comment:
I think the CRDs should either be:
* Copied here by `make crd_install` as is done for other CRD locations
* Or they should be copied at build time and instead of committing them you should just add them to .gitignore (this might be better maybe)